dreadnode/cli/platform/compose.py

```diff
@@ -0,0 +1,212 @@
+import typing as t
```

Comment:
Essentially using this file for the bonding layer between our LocalVersion object and the general docker helpers.